datasets/modeled/datacenters/dataset.json5

```
@@ -0,0 +1,118 @@
+{
+  dataset_id: "decarb_2023_datacenters",
+  dataset_type: "modeled",
+  sector_description: "Residential and commercial",
```

```suggestion
  sector_description: "Commercial",
```

Based on EIA categorization, standalone data centers are commercial.